workflow/controller/operator.go

```diff
@@ -3050,6 +3050,7 @@ func expandSequence(seq *wfv1.Sequence) ([]wfv1.Item, error) {
 			return nil, err
 		}
 	} else if seq.Count != nil {
+		seg.Count = &intstr.IntOrString{}
```

Comment:
this seems wrong?

Reply:
Yes, same as above. Should not be changed what so ever.